app/src/main/java/mozilla/lockbox/extensions/Observable+.kt

```diff
@@ -7,13 +7,30 @@
 package mozilla.lockbox.extensions
 
 import io.reactivex.Observable
+import mozilla.lockbox.LogProvider
 import mozilla.lockbox.model.ItemViewModel
 import org.mozilla.sync15.logins.ServerPassword
 
 fun <T : Any, U : T> Observable<T>.filterByType(clazz: Class<out U>): Observable<U> {
     return this.filter { t -> clazz.isInstance(t) }.map { t -> clazz.cast(t) }
 }
 
+fun <T : Any> Observable<T>.debug(): Observable<T> {
```

Comment:
Nit: `log()` `toConsole()`, `logIt()` surely?

I'm not 1000% against `debug()` however. I just like my bikesheds brown.

Reply:
I'm just copying directly what was in RxSwift :pp

Reply:
I'm just copying directly what was in RxSwift :pp